dockerfiles/agents/Dockerfile

```diff
@@ -0,0 +1,23 @@
+# Copyright (c) 2016 Codenvy, S.A.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# build:
+#   docker build -t codenvy/agents .
+#
+# use:
+#    docker run -p 9000:9000 codenvy/agents
+
+FROM alpine:3.4
+
+RUN apk update \
+    && apk add lighttpd \
+    && ln -s /data /var/www/localhost/htdocs/agent-binaries
```

Comment:
to reduce size of the image, cache of apk is usually dropped as /tmp folder : 
`rm -rf /var/cache/apk/*` and sometimes `/tmp` folder
`